cypress/e2e/homepage.cy.js

```diff
@@ -0,0 +1,5 @@
+describe('empty spec', () => {
+  it('passes', () => {
+    cy.visit('localhost:8081');
```

Comment:
Following the .env or NODE_ENV it should load the running local server start, but avoid harcoded values

Reply:
Cypress has it's own variable configs, I will work on that and push a fix